### PR TITLE
build: configure location to capture wdio artifacts

### PR DESCRIPTION
**Pull request by peternhale**
@W-16124376@
add configurable outpurDir for run artifacts
add configurable wdio debug level

### CODE REVIEW

File: test/utilities/constants.ts [hunk 1]
```diff
@@ -16,7 +14,6 @@ export const WORKSPACE_SETTING_KEYS = {
     'salesforcedx-vscode-core.push-or-deploy-on-save.preferDeployOnSave'
 };
 
-export const ONE_MINUTE = Duration.minutes(1);
-export const FIVE_MINUTES = Duration.minutes(5);
-export const TEN_MINUTES = Duration.minutes(10);
```
[the remark this answers is not in the document]
Comment (daphne-sfdc) [reply test/utilities/constants.ts:21]: They're moved to miscellaneous.ts - see Lines 206-208.

[the remark this answers is not in the document]
Comment (peternhale) [reply test/utilities/constants.ts:21]: @CristiCanizales I had to move them due to a compilation error. There error was trying to reference the constants before definition.

File: test/utilities/miscellaneous.ts [hunk 2]
```diff
@@ -20,13 +20,15 @@ export async function pause(duration: Duration = Duration.seconds(1)): Promise<v
...
 }
 
 export function debug(message: string): void {
-  if (EnvironmentSettings.getInstance().debug) {
+  if (EnvironmentSettings.getInstance().logLevel in ['debug', 'trace']) {
     const timestamp = new Date().toISOString();
-    log(`${timestamp}:${message}`);
+    console.debug(`${timestamp}:${message}`);
   }
 }
```
Comment (daphne-sfdc) [inline test/utilities/miscellaneous.ts:33]: I like this change! 😀

File: test/utilities/miscellaneous.ts [hunk 3]
```diff
@@ -71,6 +73,9 @@ export async function findElementByText(
     timeoutMsg?: string;
   }
 ): Promise<WebdriverIO.Element> {
+  if (!labelText) {
+    throw new Error('labeText must be defined');
```
Comment (daphne-sfdc) [inline test/utilities/miscellaneous.ts:77]: typo

```suggestion
    throw new Error('labelText must be defined');
```

**Comment by peternhale** [comment]
Will address in the future. currently is breaking mac tests and i don't know why.

**Pull request closed without merging**